src/ruby/meetings/spec/models/entity_spec.rb

```diff
@@ -1,37 +1,40 @@
 require 'rails_helper'
 
 RSpec.describe Entity, type: :model do
-  it{ should have_db_column(:type).of_type(:string)}
-  it{ should have_db_column(:name).of_type(:string)}
-  it{ should have_db_column(:wikipedia_entry).of_type(:string)}
-  it{ should validate_presence_of(:name)}
+  it { should have_db_column(:type).of_type(:string) }
+  it { should have_db_column(:name).of_type(:string) }
+  it { should have_db_column(:wikipedia_entry).of_type(:string) }
+  it { should validate_presence_of(:name) }
 end
 
-RSpec.describe Person,type: :model do
-  it{ should have_many(:influence_office_people)}
-  it{ should have_many(:meetings).through(:influence_office_people)}
-  it{ should have_many(:hospitalities).through(:influence_office_people)}
-  it{ should have_many(:gifts).through(:influence_office_people)}
-  it{ should have_many(:travels).through(:influence_office_people)}
+RSpec.describe Person, type: :model do
+  it { should have_many(:influence_office_people) }
+  it { should have_many(:meetings).through(:influence_office_people) }
+  it { should have_many(:hospitalities).through(:influence_office_people) }
+  it { should have_many(:gifts).through(:influence_office_people) }
+  it { should have_many(:travels).through(:influence_office_people) }
+  it { should_not validate_uniqueness_of(:name) }
 end
 
-RSpec.describe Organisation,type: :model do
-  it{ should have_many(:influence_organisation_people)}
-  it{ should have_many(:meetings).through(:influence_organisation_people)}
-  it{ should have_many(:hospitalities).through(:influence_organisation_people)}
-  it{ should have_many(:gifts).through(:influence_organisation_people)}
-  it{ should have_many(:travels).through(:influence_organisation_people)}
+RSpec.describe Organisation, type: :model do
+  it { should have_many(:influence_organisation_people) }
+  it { should have_many(:meetings).through(:influence_organisation_people) }
+  it { should have_many(:hospitalities).through(:influence_organisation_people) }
+  it { should have_many(:gifts).through(:influence_organisation_people) }
+  it { should have_many(:travels).through(:influence_organisation_people) }
+  it { should validate_uniqueness_of(:name) }
 end
 
-RSpec.describe GovernmentOffice,type: :model do
-  it{ should have_many(:influence_government_office_people)}
-  it{ should have_many(:meetings).through(:influence_government_office_people)}
-  it{ should have_many(:hospitalities).through(:influence_government_office_people)}
-  it{ should have_many(:gifts).through(:influence_government_office_people)}
-  it{ should have_many(:travels).through(:influence_government_office_people)}
+RSpec.describe GovernmentOffice, type: :model do
+  it { should have_many(:influence_government_office_people) }
+  it { should have_many(:meetings).through(:influence_government_office_people) }
+  it { should have_many(:hospitalities).through(:influence_government_office_people) }
```

Comment:
I think in this context, leaving the long line is an acceptable compromise - it's better than the alternatives.